pgcrypto/lookups.py

```diff
@@ -41,3 +44,101 @@ class HMACLookup(HashLookupBase):
     the field's value.
     """
     encrypt_sql = HMAC_SQL
+
+
+class DateLookupBase(Lookup):
+    """Base class for Date lookups."""
+    lookup_name = None  # Set in subclasses
+    operator = None  # Set in subclasses
+
+    def __init__(self, lhs, rhs):
+        """Implementing an abstract class."""
+        super(DateLookupBase, self).__init__(lhs, rhs)  # pragma: no cover
```

Comment:
As we do nothing in `__init__` except calling `super`, we don't need this method, you can remove it.